config/initializers/statsd.rb

```diff
@@ -0,0 +1,28 @@
+Notifier.extend StatsD::Instrument
+Notifier.statsd_count_success :perform, 'Webhook.perform.success'
+
+Indexer.extend StatsD::Instrument
+Indexer.statsd_count_success :perform, 'Indexer.perform.success'
+Indexer.statsd_measure :perform, 'Indexer.perform'
+
+SearchesController.extend StatsD::Instrument
+SearchesController.statsd_measure :show, 'SearchesController.show'
+
+ActiveSupport::Notifications.subscribe /process_action.action_controller/ do |*args|
+  event = ActiveSupport::Notifications::Event.new(*args)
+  event.payload[:format] = event.payload[:format] || 'all'
+  event.payload[:format] = 'all' if event.payload[:format] == '*/*'
+  status = event.payload[:status]
+  ActiveSupport::Notifications.instrument :performance, event.payload.merge(statsd_method: :measure, measurement: 'total_duration', value: event.duration)
+  ActiveSupport::Notifications.instrument :performance, event.payload.merge(statsd_method: :measure, measurement: 'db_time', value: event.payload[:db_runtime])
+  ActiveSupport::Notifications.instrument :performance, event.payload.merge(statsd_method: :measure, measurement: 'view_time', value: event.payload[:view_runtime])
+  ActiveSupport::Notifications.instrument :performance, event.payload.merge(measurement: "status.#{status}")
```

Comment:
why do we need to send this to another Notification? and not just log it in here?